app/config/collections/platform.php

```diff
@@ -0,0 +1,1534 @@
+                'required' => true,
+                'default' => null,
+                'array' => false,
+                'filters' => []
+            ],
+            [
+                '$id' => ID::custom('providerBranch'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => true,
+                'default' => null,
+                'array' => false,
+                'filters' => []
+            ],
+        ],
+        'indexes' => [
+            [
+                '$id' => ID::custom('_key_installationId'),
+                'type' => Database::INDEX_KEY,
+                'attributes' => ['installationId'],
+                'lengths' => [Database::LENGTH_KEY],
+                'orders' => [Database::ORDER_ASC],
+            ],
+            [
+                '$id' => ID::custom('_key_installationInternalId'),
+                'type' => Database::INDEX_KEY,
+                'attributes' => ['installationInternalId'],
+                'lengths' => [Database::LENGTH_KEY],
+                'orders' => [Database::ORDER_ASC],
+            ],
+            [
+                '$id' => ID::custom('_key_projectInternalId'),
+                'type' => Database::INDEX_KEY,
+                'attributes' => ['projectInternalId'],
+                'lengths' => [Database::LENGTH_KEY],
+                'orders' => [Database::ORDER_ASC],
+            ],
+            [
+                '$id' => ID::custom('_key_projectId'),
+                'type' => Database::INDEX_KEY,
+                'attributes' => ['projectId'],
+                'lengths' => [Database::LENGTH_KEY],
+                'orders' => [Database::ORDER_ASC],
+            ],
+            [
+                '$id' => ID::custom('_key_providerRepositoryId'),
+                'type' => Database::INDEX_KEY,
+                'attributes' => ['providerRepositoryId'],
+                'lengths' => [Database::LENGTH_KEY],
+                'orders' => [Database::ORDER_ASC],
+            ],
+            [
+                '$id' => ID::custom('_key_providerPullRequestId'),
+                'type' => Database::INDEX_KEY,
+                'attributes' => ['providerPullRequestId'],
+                'lengths' => [Database::LENGTH_KEY],
+                'orders' => [Database::ORDER_ASC],
+            ],
+            [
+                '$id' => ID::custom('_key_providerBranch'),
+                'type' => Database::INDEX_KEY,
+                'attributes' => ['providerBranch'],
+                'lengths' => [Database::LENGTH_KEY],
+                'orders' => [Database::ORDER_ASC],
+            ],
+        ],
+    ],
+
+    'vcsCommentLocks' => [
+        '$collection' => ID::custom(Database::METADATA),
+        '$id' => ID::custom('vcsCommentLocks'),
+        'name' => 'vcsCommentLocks',
+        'attributes' => [],
+        'indexes' => []
+    ]
```

Comment:
are you sure this collection has no attributes?

Reply:
Doesn't seem like - https://github.com/appwrite/appwrite/blob/7ecba8c0f282567911904de266ec1f0b4e0546a6/app/config/collections.php#L5701C5-L5707C7

Not sure if this is a leftover from previous implementations.